src/uct/ib/dc/verbs/dc_verbs.c

```diff
@@ -561,20 +561,27 @@ ucs_status_t uct_dc_verbs_ep_flush(uct_ep_h tl_ep, unsigned flags, uct_completio
 }
 
 ucs_status_t uct_dc_verbs_iface_create_ah(uct_dc_iface_t *dc_iface, uint16_t lid,
-                                          struct ibv_ah **ah_p)
+                                          union ibv_gid *gid, struct ibv_ah **ah_p)
 {
-    struct ibv_ah_attr ah_attr;
+    struct ibv_ah_attr ah_attr = {};
     struct ibv_ah *ah;
     uct_ib_iface_t *iface = &dc_iface->super.super;
 
-    /* TODO: GRH, path_bits, etc */
     ah_attr.sl            = iface->config.sl;
     ah_attr.src_path_bits = iface->path_bits[0];
     ah_attr.dlid          = lid | ah_attr.src_path_bits;
     ah_attr.port_num      = iface->config.port_num;
-    ah_attr.is_global     = 0;
     ah_attr.static_rate   = 0;
 
+    if ((iface->addr_type != UCT_IB_ADDRESS_TYPE_GLOBAL) &&
+        (iface->gid.global.subnet_prefix == gid->global.subnet_prefix)) {
+        ah_attr.is_global     = 0;
+    } else {
+        ah_attr.is_global     = 1;
```

Comment:
Can you please align assignments by column?

Reply:
fixed